app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -7,17 +7,17 @@ import notesApi from "apis/notes";
 const DeleteAlert = ({
   refetch,
   onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
+  selectedNoteId,
+  setSelectedNoteId,
 }) => {
   const [deleting, setDeleting] = useState(false);
 
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ id: selectedNoteId });
```

Comment:
In the backend, does the `destroy` action accept a param called `id`? Also does the `notesApi.destory` API connector accept a key called `id`? Did you simply change the name of the key just because I had pointed out in the review about the mismatch between key name and value, without even verifying the working of it?

Reply:
I will correct this mistake. I believe changing this to `await notesApi.destroy({ ids: selectedNoteId });` will make things work, without having to make changes to other files. But will it be a best-practices violation to pass a single note ID with a key of `ids`?

Another solution I can think of is to pass the singular `selectedNoteId` to the `selectedNoteIds` prop of the `DeleteAlert` component, making it possible to keep the API call as `await notesApi.destroy({ ids: selectedNoteIds });` itself, like: 

```
<DeleteAlert
       refetch={fetchNotes}
       selectedNoteIds={selectedNoteId}
       setSelectedNoteIds={setSelectedNoteId}
       onClose={() => setShowDeleteAlert(false)}
/>
```

Can you please suggest which one would be the best? Thanks, in advance!

Reply:
@anil-anand All you have to do is pass an array of IDs. The array can have 1 ID or multiple IDs.

```js
await notesApi.destroy({ ids: [selectedNoteId] });
```